Intersect (Core)/Extensions/VariableModExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using System.Reflection;
+using Intersect.Attributes;
+using Intersect.Enums;
+
+namespace Intersect.Extensions;
+public static class VariableModExtensions
+{
+    public static VariableType GetRelatedVariableType(this VariableMod value)
+    {
+        if (!Enum.IsDefined(typeof(VariableMod), value))
+        {
+            throw new ArgumentException($"{nameof(value)} is not a valid VariableMod enum");
```

Comment:
There's a `parameterName` parameter after the message

Reply:
I think there also might be a value parameter after that to pass `value` into so it shows up in logs